.github/workflows/build.yml

```
@@ -78,13 +78,39 @@ jobs:
       - name: Gradle build
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: --refresh-dependencies --stacktrace --scan clean build
+          arguments: --refresh-dependencies --stacktrace --scan clean build -x spotlessCheck
       - name: SonarCloud analysis
         if: ${{ matrix.os == 'ubuntu' }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: "61ab2579215aa8a0024a2f9368fc1298fdecfd18"
         run: ./gradlew jacocoTestReport sonarqube --stacktrace -i
+      - name: Test report
+        uses: mikepenz/action-junit-report@v2
+        if: always() # always run even if the previous step fails
+        with:
+          check_name: "Test report for Java ${{ matrix.java }} on ${{ matrix.os }}"
+          report_paths: '**/build/test-results/test/TEST-*.xml'
+
+  # Code format check
+  code-format-check:
+    runs-on: ubuntu-latest
+    needs: validation
+    timeout-minutes: 15
+    name: Code format check
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
```

Meanwhile we use Java 11:

```suggestion
          java-version: 11
```